client/components/foldable-card/index.jsx

```diff
@@ -85,10 +87,10 @@ var FoldableCard = React.createClass( {
 			);
 		}
 		if ( this.props.children ) {
-			let iconSize = 24;
+			const iconSize = 24;
 			return (
 				<button disabled={ this.props.disabled } className="foldable-card__action foldable-card__expand" onClick={ clickAction }>
-					<span className="screen-reader-text">{ this.translate( 'More' ) }</span>
+					<span className="screen-reader-text">{ this.props.screenReaderText }</span>
```

Comment:
Should we just do `this.props.screenReaderText || this.translate( 'More' )` so it's only needed as a prop when you want to change it?